src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -19,9 +19,19 @@ def cli_redis_import_method(client, resource_group_name, name, file_format, file
     parameters = ImportRDBParameters(files, file_format)
     return client.import_method(resource_group_name, name, files, parameters)
 
-def cli_redis_update_settings(client, resource_group_name, name, redis_configuration):
+def cli_redis_update_settings(client, resource_group_name, name, redis_configuration=None, # pylint:disable=too-many-arguments
+                              sku_name=None, sku_family=None, sku_capacity=None):
+    if sku_name != None or sku_family != None or sku_capacity != None:
```

Comment:
Thanks for the recommendation. Will update it. About renaming the command name, wouldnt changing the name break existing customers who use this for automations? Or since the product is in its early stages it's okay to do that? I'll be very willing to change the command if you can confirm its okay